src/common/services/api/radioStations.service.js

```diff
@@ -19,14 +19,14 @@ class RadioStations {
     this.$filter = $filter
   }
 
-  getRadioStations(radioStationApiUrl, postalCode, radioStationRadius) {
+  getRadioStations (radioStationApiUrl, postalCode, radioStationRadius) {
```

Comment:
This commit (dbc64fc294bf37dfd8d782be6c8ab040f84c5fa4) should be squashed with 92d24a86535a5695596ec1c476884b4e0006a276